packages/content-management/content_insights/content_insights_public/src/components/views_stats/views_stats.tsx

```diff
@@ -86,7 +86,7 @@ const NoViewsTip = () => (
     content={
       <FormattedMessage
         id="contentManagement.contentEditor.viewsStats.noViewsTip"
-        defaultMessage="Views are counted everytime someone opens a dashboard (after version 8.16)"
+        defaultMessage="Views are counted every time someone opens a dashboard (after 11-2024)"
```

Comment:
I think there is an edge-case that at `11-2024` this deployment could have been on a version before 8.16., isn't it? 

Maybe it is worth propagating `isServerless` here and not showing the version if it is serverless? Or just removing version/date here completely, same for user_missing_tip

Reply:
Yep, hence my comment above

Reply:
I do agree we probably would actually have a deployment containing this before this date, at the same time we'd briefly talked about how we weren't sure if it be worth the effort to propagate the `isServerless` value here and I still don't think it is. I think it's more complexity for this interface, I'm not opposed to removing it entirely and ensuring that this feature gets a mention in the release note for 8.16 so that users can found out if they so wish to learn when this feature became available